Know_Base/Git_notes.md

```diff
@@ -38,3 +38,17 @@ Git Commands
 
     $ git status
 
+Branching...
+
+  9. Creating new Barnch
+	
+    $ git branch branch_name
+
+  10. See the branch list
+
+    $ git branch --list
+
```

Comment:
remove extra spaces
you nedd to use some functions sqrt or us if insted swtch or while or for